deploy.sh

```diff
@@ -60,6 +60,13 @@ else
     fi
 fi
 
+aws sts get-caller-identity
+result=$?
+if [ $result != 0 ]; then
+    echo "AWS ACCESS KEY or SECRET isn't set. Check out our doc to understand what you're missing https://bitops.sh/examples/#docker-run-examples"
+    exit $result
```

Comment:
👍

Reply:
Sorry does something need to be resolved here?

Reply:
No, it's good. Just providing feedback on the part I liked.